dashboard/v1.1/INSTALL.md

```diff
@@ -0,0 +1,5 @@
+### Installation Instructions
+
+Please follow MakeFile in the root directory of the project to install the UI.
```

Comment:
`MakeFile` => `Makefile`

also, the makefile needs an extension. Launching the new UI is not supported by make as of now. Ca you do this? Would make it easier to test and run the new UI.

Reply:
Sure.